core/web/log_controller.go

```diff
@@ -17,8 +17,11 @@ type LogController struct {
 }
 
 type LogPatchRequest struct {
-	Level      string `json:"level"`
-	SqlEnabled *bool  `json:"sqlEnabled"`
+	Level        string `json:"level"`
+	Filter       string `json:"filter"`
+	SqlEnabled   *bool  `json:"sqlEnabled"`
+	ServiceName  string `json:"serviceName"`
```

Comment:
hmm we'd probably want to do a batch update of these things so we don't send a request per service - maybe a slice of `type ServiceLogLevel [2]string` here instead?

Reply:
updated to support a batch update, can pass through a list now of services & log levels, will add some tests around this and should be explicit in docs